packages/react-devtools-shared/src/__tests__/transform-react-version-pragma-test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+const semver = require('semver');
+
+let shouldPass;
+let isFocused;
+describe('transform-react-version-pragma', () => {
+  // eslint-disable-next-line no-unused-vars
+  const _test_react_version = (range, testName, cb) => {
+    test(testName, (...args) => {
+      shouldPass = semver.satisfies('18.0.0', range);
+      return cb(...args);
+    });
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const _test_react_version_focus = (range, testName, cb) => {
+    test(testName, (...args) => {
+      shouldPass = semver.satisfies('18.0.0', range);
+      isFocused = true;
+      return cb(...args);
+    });
+  };
```

Comment:
So the transform below converts the pragmas to call these two functions, right? How will this work in a normal test (one that doesn't define `_test_react_version` or `_test_react_version_focus`)?

Reply:
I think I'm not understanding how the pragmas are supposed to work for regular tests then? Sorry maybe I'm being dumb

Reply:
Like the React regression tests you're going to write– the ones you're adding this pragma to support.

Reply:
Sorry, I'm still a bit confused by your question. Could you reword it?

Reply:
How do I use the new pragma syntax to write the _actual_ DevTools+React regression tests (not just ones that test the Babel transform)?

Do I literally need to copy-paste these functions into every test file so I can use them? (I don't think that's the case but also– I'm not sure how it's supposed to work.)

Reply:
Oh, no you don't. We add `global._test_react_version` and `global._test_react_version_focus` in the `setupEnv` so normally we would use those. I'm just overwriting them here to test that it's working.